src/Documentation/resources/Best_Practices.md

```diff
@@ -0,0 +1,422 @@
+# Best Practices
+
+Orleans was designed in 2014 with the intention of removing low-level
```

Comment:
Orleans has been designed and built much earlier. It was running in production in late 2011. Also, the intent had nothing to do with improving the actor model per se. I think it's better to rephrase the into to something like the following.

"Orleans was built with the goal to greatly simplify building of distributed scalable applications, especially for the cloud. Orleans invented the [Virtual Actor Model](https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/Orleans-MSR-TR-2014-41.pdf) as an evolution of the Actor Model optimized for the cloud scenarios.

Grains (virtual actors) are the base building blocks of an Orleans-based application. They encapsulate state and behavior of application entities and maintain their lifecycle, The programming model of Orleans and the characteristics of its runtime fit some types of applications better than others. This document is intended to capture some of the tried and proven application patterns that work well in Orleans."

Reply:
Corrected.